mobile/lib/features/wallet/create_invoice_screen.dart

```diff
@@ -222,3 +230,44 @@ class _CreateInvoiceScreenState extends State<CreateInvoiceScreen> {
     );
   }
 }
+
+void showNoLiquidityDialog() {
+  showDialog(
+      context: rootNavigatorKey.currentContext!,
+      builder: (context) => AlertDialog(
+              title: const Text("No liquidity in the LSP."),
+              content: RichText(
+                text: TextSpan(
+                  style: DefaultTextStyle.of(context).style,
```

Comment:
🎨 The default text style looks very odd. Please change to something that is more consistent to our otherwise used styles.